### PR TITLE
Add source maps to VS launcher and Dockerfile

### PR DESCRIPTION
**Pull request by marcos-arranz**
Adding source maps as a env variable when run in VS Code and in Dockers container defintion, so debugging and error reporting is easier.

### CODE REVIEW

**Review by jsfr — APPROVED** [review]

File: .vscode/launch.json [hunk 1]
```diff
@@ -4,9 +4,13 @@
   // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
   "version": "0.2.0",
   "configurations": [
+
```
Comment (andersfischernielsen) [inline .vscode/launch.json:7]:
```suggestion
```

File: Dockerfile [hunk 2]
```diff
@@ -8,8 +8,8 @@ COPY .swcrc ./.swcrc
 
 RUN npm install -g pnpm@7
 RUN pnpm install --frozen-lockfile \
-  && pnpm build \
-  && pnpm install --frozen-lockfile --prod
+    && pnpm build \
+    && pnpm install --frozen-lockfile --prod
```
Comment (andersfischernielsen) [inline Dockerfile:12]:
```suggestion
  && pnpm build \
  && pnpm install --frozen-lockfile --prod
```

File: package.json [hunk 3]
```diff
@@ -57,4 +57,4 @@
     "ts-jest": "29.0.3",
     "typescript": "4.8.4"
   }
-}
+}
```
Comment (andersfischernielsen) [inline package.json:60]:
```suggestion
}

```

**Review by andersfischernielsen — APPROVED** [review]
LGTM! I assume sourcemaps have no performance penalty when running via the container image? 😄

**Comment by marcos-arranz** [comment]
> LGTM! I assume sourcemaps have no performance penalty when running via the container image? 😄

Not significant. It's not like we file distributor has to be performant critical 😄

**Pull request merged**